vendor/vendor-ios/src/main/kotlin/com/malinskiy/marathon/ios/IOSTestParser.kt

```diff
@@ -20,6 +21,11 @@ class IOSTestParser(private val vendorConfiguration: VendorConfiguration.IOSConf
      *  marked as skipped in `xctestrun` file.
      */
     override suspend fun extract(): List<Test> {
+
+        if (vendorConfiguration.xcTestRunnerTag.isNullOrEmpty()) {
+            logger.warn { "[iOS] Did not find `xcTestRunnerTag` in iOS Vendor Config. This will result in running all tests" }
```

Comment:
The tag filtering doesn't have anything to do with the responsibility of the parser, this probably doesn't belong here